test/testcafe/spec/Success_spec.js

```diff
@@ -8,10 +8,17 @@ const mock = RequestMock()
   .onRequestTo('http://localhost:3000/idp/idx/introspect')
   .respond(identify)
   .onRequestTo('http://localhost:3000/idp/idx/identify')
-  .respond(success);
+  .respond(success)
+  .onRequestTo(/^http:\/\/localhost:3000\/app\/UserHome.*/)
+  .respond(`
+        <h1 id="mock-user-dashboard-title">Mock User Dashboard</h1>
+        <h2>Query parameters</h2>
+        <a href="/">Back to Login</a>
+  `);
```

Comment:
Is this done to prevent the page redirection? Just wondering if this is necessary because another spec I worked on had a similar redirect (want to know if I should add this same response).

Reply:
This is actually to give the browser something to render upon redirection. The existing test relies on a Dyson mock but in v3 we don't have that server running so instead it's simpler to just inline it into the TestCafe mocks.

Reply:
Ah that makes sense. Ty!